packages/web-workers/src/messenger/i-frame/remote-origin.ts

```diff
@@ -0,0 +1,8 @@
+import {createIframeWorkerMessenger} from './create-messenger';
+
+export function createRemoteOriginIframeMessenger(url: URL, iframeSrc: string) {
```

Comment:
I always get a bit worried to see >1 positional argument when all the arguments are primitive (or primitive-like, like `URL`s). I'd probably wrap up that second argument in an options object.